### ISSUE TITLE
[$75] Screen Reader: Many Pages: The embedded links cannot be focused and activated

### ISSUE THREAD

**Issue by olenamihno** [Expensify/App#77553]
## Action Performed:
Prerequisites:
- the user is not signed in

1. Using iOS VoiceOver, open the Expensify app
2. Swipe to the 'Phone and email' field
3. Type the valid email address and double-tap to activate
4. Swipe and try to focus the 'Terms of Services' and 'Privacy' links

Other Occurrences:
1. On About: the links Terms of Service and Privacy
2. On Troubleshoot: 'submit a bug'

## Expected Result:
Each link should be focused separately and announced with a link role and its visible name.

## Actual Result:
The embedded links 'Terms of Services' and 'Privacy' cannot be focused and activated.

## Area issue was found in:
Sign in

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Ensure the links are not grouped with the surrounding text, so the user can focus and activate each link.
https://reactnative.dev/docs/accessibility#accessible

## Environment:
Apple iPhone 11 - iOS 17.5.1, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos

<img width="828" height="1792" alt="Image" src="https://github.com/user-attachments/assets/a5ccae26-16e0-4b2d-a5f4-08061ca96348" />

https://github.com/user-attachments/assets/d0209cc7-a9ab-4523-8fbb-9421b5caa1d0

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** No
**Logs:** https://stackoverflow.com/c/expensify/questions/4856  NA
**Notes/Photos/Videos:** Any additional supporting documentation  Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7005417
**Issue reported by:** NA
**Slack conversation:** NA
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~022009685103363347151</li>
        <li>Upwork Job ID: 2009685103363347151</li>
        <li>Last Price Increase: 2026-01-09</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @grgia</details>

**Comment by parasharrajat** [comment]
@TaduJR Will it render as `anchor` tag on web?

**Comment by parasharrajat** [comment]
Bump @grgia

**Comment by TaduJR** [comment]
Offer Accepted @mallenexpensify 

Thanks!
